english/coffee/hollywood/hollywood.md

```diff
@@ -0,0 +1 @@
+After writing, what you believe to be to be, a best selling novel you decided to try and market the book right and head to Hollywood. The only choice now is how to get there. You’re afraid of flying and don’t care much for public transportation. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.